macrosynergy/pnl/historic_portfolio_volatility.py

```diff
@@ -94,33 +97,56 @@ def flat_std(x: np.ndarray, remove_zeros: bool = True) -> float:
     return mabs
 
 
-def _single_calc(
-    row,
+def _rolling_window_calc(
+    real_date_pdt: pd.Timestamp,
     ticker_df: pd.DataFrame,
     lback_periods: int,
     nan_tolerance: float,
     roll_func: callable,
     remove_zeros: bool,
     weights: Optional[np.ndarray] = None,
```

Comment:
As far as I can see neither of these three options are currently being used?

Reply:
they are passed in line 273 (declared as `_args` , updated in 246, 248 ...)

Reply:
Currently, they are not used as the univariate volatility calc. is commented out (L135->L147).